packages/framework/src/Component/Cron/CronModule.php

```diff
@@ -121,7 +121,7 @@ public function suspend()
     /**
      * @return bool
      */
-    public function isEnabled(): bool
+    public function isEnabled()
```

Comment:
I think that `is` and `has` methods could have an exception as this naming is only used for the booleans.

Reply:
boolean types are now ignored